dev/com.ibm.ws.security.wim.adapter.ldap_fat.krb5/fat/src/com/ibm/ws/security/wim/adapter/ldap/fat/krb5/FATSuite.java

```diff
@@ -0,0 +1,38 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.security.wim.adapter.ldap.fat.krb5;
+
+import org.junit.ClassRule;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import componenttest.custom.junit.runner.AlwaysPassesTest;
+import componenttest.rules.repeater.FeatureReplacementAction;
+import componenttest.rules.repeater.JakartaEE9Action;
+import componenttest.rules.repeater.RepeatTests;
+
+@RunWith(Suite.class)
+@SuiteClasses({
+                AlwaysPassesTest.class
+})
+public class FATSuite {
+
+    // Using the RepeatTests @ClassRule will cause all tests to be run three times.
+    // 1) without any modifications
+    // 2) [FULL mode only] again with all features upgraded to their EE8 equivalents
+    // 3) [FULL mode only] again with all features _and applications_ upgrade to Jakarta EE 9 equivalents
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
```

Comment:
Get rid of this. I don't think there is any value added to running with EE8 or EE9. We are testing the registry, not any EE components.

Reply:
Done